<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Level Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>19</y>
      <width>791</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>791</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Modify...</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>20</y>
           <width>46</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>155</x>
           <y>20</y>
           <width>46</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>275</x>
           <y>20</y>
           <width>46</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Traslation</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>20</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Current Position</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelCurrentPos">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>35</y>
           <width>231</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>[x = 0; y = 0; z = 0]</string>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterScaleX">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterScaleY">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterScaleZ">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterRotX">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterRotY">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterRotZ">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterTraY">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>60</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterTraZ">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>80</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QwtCounter" name="CounterTraX">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>40</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="basicstep" stdset="0">
          <double>0.200000000000000</double>
         </property>
         <property name="minValue">
          <double>-9999.000000000000000</double>
         </property>
         <property name="maxValue">
          <double>9999.000000000000000</double>
         </property>
         <property name="stepButton2">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>60</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Current Scale</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelCurrentScale">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>75</y>
           <width>231</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>[x = 0; y = 0; z = 0]</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_Animation">
        <attribute name="title">
         <string>Animation</string>
        </attribute>
        <widget class="QComboBox" name="comboBox_Animation">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>199</width>
      <height>701</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Items List</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>View Scene</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Scene"/>
    <addaction name="actionSave_Scene"/>
    <addaction name="actionLoad_Scene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTerrain">
    <property name="title">
     <string>Terrain</string>
    </property>
    <addaction name="actionGenerate_Random"/>
    <addaction name="actionSava_as_HMap"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Texture"/>
   </widget>
   <widget class="QMenu" name="menuModels">
    <property name="title">
     <string>Models</string>
    </property>
    <addaction name="actionAddEntity"/>
    <addaction name="actionDeleteEntity"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Light"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnline_Guide"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionEnable_Shadow"/>
    <addaction name="actionEnable_Animation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTerrain"/>
   <addaction name="menuModels"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Scene">
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionSave_Scene">
   <property name="text">
    <string>Save Scene</string>
   </property>
  </action>
  <action name="actionLoad_Scene">
   <property name="text">
    <string>Load Scene</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGenerate_Random">
   <property name="text">
    <string>Generate Random</string>
   </property>
  </action>
  <action name="actionSava_as_HMap">
   <property name="text">
    <string>Sava as HMap</string>
   </property>
  </action>
  <action name="actionLoad_Texture">
   <property name="text">
    <string>Load Texture</string>
   </property>
  </action>
  <action name="actionAddEntity">
   <property name="text">
    <string>Add Entity</string>
   </property>
  </action>
  <action name="actionDeleteEntity">
   <property name="text">
    <string>Delete Entity</string>
   </property>
  </action>
  <action name="actionOnline_Guide">
   <property name="text">
    <string>Online Guide</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEnable_Shadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Shadow</string>
   </property>
  </action>
  <action name="actionEnable_Animation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Animation</string>
   </property>
  </action>
  <action name="actionEnable_Light">
   <property name="text">
    <string>Enable Light</string>
   </property>
  </action>
  <action name="actionCreate_Light">
   <property name="text">
    <string>Create Light</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
